### ISSUE TITLE
Can't find extension in status bar

### ISSUE THREAD

**Issue by Shaokun-X** [onlyutkarsh/git-config-user-profiles#48]
**Describe the bug**
In `1.1.1`, can't find the extension in status bar

**Screenshots**
![image](https://user-images.githubusercontent.com/25054748/130329301-e7f27f49-4d82-414f-990e-e15ca79d9398.png)

**Environment**
```
Version: 1.59.1 (user setup)
Commit: 3866c3553be8b268c8a7f8c0482c0c0177aa8bfa
Date: 2021-08-19T11:56:46.957Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Windows_NT x64 10.0.19041
```

**Comment by onlyutkarsh** [comment]
thanks @Shaokun-X  - I have observed this behaviour as well. Not sure what is causing this - I just think extension is not getting activated for some reason. There are no errors in the logs.

![ScreenShot-21-08-2021 - 6 00 38](https://user-images.githubusercontent.com/2685029/130329470-2bf7af99-89ec-4cdf-a37c-16cecc686cd3.jpg)

**Comment by Shaokun-X** [comment]
@onlyutkarsh I think it is some deployment issue, I checked VSC dev console and found this:
```
Activating extension 'onlyutkarsh.git-config-user-profiles' failed: Cannot find module 'c:\Users\User\.vscode\extensions\onlyutkarsh.git-config-user-profiles-1.1.1\out\extension.js'
Require stack:
- d:\Microsoft VS Code\resources\app\out\vs\loader.js
- d:\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- d:\Microsoft VS Code\resources\app\out\bootstrap-fork.js.
```

So I headed to the folder and didn't see the built `extension.js`:
```
onlyutkarsh.git-config-user-profiles-1.1.1
│  .vsixmanifest
│  CHANGELOG.md
│  jest.config.js
│  LICENSE.txt
│  package.json
│  README.md
│
└─images
    │  icon.png
    │
    └─marketplace
            demo.gif
            outputwindow.png
            profile-picker.png
            repo-in-sync.png
            repo-not-in-sync.png
            statusbar-picker.png
            statusbar.png
```

**Comment by onlyutkarsh** [comment]
Ah! that must be the issue. thanks for troubleshooting. Let me come back..

**Comment by Shaokun-X** [comment]
> Ah! that must be the issue. thanks for troubleshooting. Let me come back..

No problem, maybe you can change the deploy script so that every time it is sure to be built. Really like the extension, thank you for the work!

**Comment by onlyutkarsh** [comment]
@Shaokun-X thanks again for your time in troubleshooting. This should be fixed now in v1.1.2.
